sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/CreateMessageBatchOptions.cs

```diff
@@ -40,6 +43,30 @@ public class CreateMessageBatchOptions
             }
         }
 
+        /// <summary>
+        ///   The maximum size to allow for a single batch of messages, in number of items.
+        /// </summary>
+        ///
+        /// <value>
+        ///   The desired limit, in number of items, for the size of the associated service bus message batch.  If <c>null</c>,
+        ///   the maximum size allowed by the active transport will be used.
+        /// </value>
+        ///
+        internal int? MaxSizeInNumberOfItems
+        {
+            get => _maxSizeInNumberOfItems;
+
+            set
+            {
+                if (value.HasValue)
+                {
+                    Argument.AssertInRange(value.Value, 1, 4500, nameof(MaxSizeInNumberOfItems));
```

Comment:
I doubt having a batch size of one ever makes sense yet I had to get started with something.